packages/app-defaults/catalog-info.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: backstage-app-defaults
+  title: '@backstage/app-defaults'
+  description: Provides the default wiring of a Backstage App
+spec:
+  lifecycle: experimental
+  type: backstage-package
+  backstage_role: web-library
```

Comment:
I'm wondering about:

1. type `backstage-package` vs `library` as in the default model docs. The extension field `backstage_role` could be enough indication. 

2. `backstage_role` seems not to follow the camelCase naming pattern. I would go for ` backstageRole` for consistency reasons.

Reply:
Great call-out re: `backstage_role`!  I'll update.

On `backstage-package` vs. `library`: we definitely considered this.

Our hesitation with `library` is that, although open source plugins resemble what are typically modeled as library components (e.g. they are packaged code, distributed to some kind of registry, with versioned releases), innersource plugins do not map well to this paradigm (they are versionless pieces of code built and deployed directly alongside an existing service--an organization's Backstage instance).  Maybe less important, but usage patterns of libraries are also different from these backstage packages: libraries you might expect to see used across a variety of systems in a variety of contexts, while Backstage packages are almost exclusively used within Backstage (and often only used by a particular plugin).

Hope is that this will simplify entity pages for these components (just leverage `isComponentType()` and let it sit alongside other types without having to break down a particular type further), and that it might simplify targeting these types of components for the purposes of evaluating tech health.